packages/common/src/utils/submissionUtils.ts

```diff
@@ -55,14 +56,18 @@ const utils = (dependencies: Dependencies) => {
 				Object.keys(schemaErrors).length > 0 ? SUBMISSION_STATE.INVALID : SUBMISSION_STATE.VALID;
 			if (isNumber(idActiveSubmission)) {
 				// Update with new data
-				updatedSubmission = await submissionRepo.update(_.toNumber(idActiveSubmission), {
+				const resultUpdate = await submissionRepo.update(_.toNumber(idActiveSubmission), {
 					data: entityMap,
 					state: newStateSubmission,
 					organization,
 					dictionaryId,
 					updatedBy: userName,
 					errors: schemaErrors,
 				});
+				if (!resultUpdate) throw new InternalServerError();
```

Comment:
Please add brackets around the `if` block.

Should there be a message given to the internal server error? Or some logging?

Reply:
based on the whole purpose of this function to create/update an Active Submission in database it makes more sense to throw an error than just a log